### PR TITLE
View stories' source code

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/zakodium-oss/react-science/issues/533

### CODE REVIEW

**Comment by wadjih-bencheikh18** [comment]
the webkit test is not stable (It works in local after updating playwright)
should we add retries in playwright?

**Comment by targos** [comment]
> the webkit test is not stable (It works in local after updating playwright) should we add retries in playwright?

It would be nice if we can understand and fix the root cause. As a last resort, we would add retries.

I think the first thing to do should be to upload playwright reports when tests fail, like we do already in cheminfo/nmrium

File: package.json [hunk 1]
```diff
@@ -103,6 +103,7 @@
     "@playwright/experimental-ct-react": "^1.36.2",
     "@playwright/test": "^1.36.2",
     "@storybook/addon-essentials": "7.2.1",
+    "@storybook/addon-storysource": "^7.2.2",
```
Comment (targos) [inline package.json:106]:
```suggestion
    "@storybook/addon-storysource": "7.2.1",
```

Please revert the changes and redo with a fixed version (same for all storybook packages). Storybook doesn't like it when we mix up package versions.

**Review by targos — APPROVED** [review]
